code/datums/uplink_item.dm

```diff
@@ -649,7 +649,16 @@ var/list/uplink_items = list()
 	Nanotrasen crewmembers are trained to report red space suit sightings; these suits in particular are known to drive employees into a panic."
 	item = /obj/item/clothing/suit/space/hardsuit/syndi
 	cost = 8
-	excludefrom = list(/datum/game_mode/gang)
+	excludefrom = list(/datum/game_mode/nuclear,/datum/game_mode/gang)
+
+/datum/uplink_item/device_tools/hardsuit/nuke
+	name = "Additional Blood-Red Hardsuit"
+	desc = "The feared suit of a syndicate nuclear agent. Features slightly better armoring and a built in jetpack that runs off standard atmospheric tanks. \
+	When the built in helmet is deployed your identity will be protected, even in death, as the suit cannot be removed by outside forces. Toggling the suit into combat mode \
+	will allow you all the mobility of a loose fitting uniform without sacrificing armoring. Additionally the suit is collapsible, small enough to fit within a backpack. \
+	Nanotrasen crewmembers are trained to report red space suit sightings; these suits in particular are known to drive employees into a panic."
+	cost = 4
+	gamemodes = list(/datum/game_mode/nuclear)
```

Comment:
You need `excludefrom = list()` above or below this, as otherwise it'll inherit the exclusion from /datum/uplink_item/device_tools/hardsuit